cli/src/wallet.rs

```diff
@@ -671,6 +676,7 @@ pub fn process_transfer(
     fee_payer: SignerIndex,
     derived_address_seed: Option<String>,
     derived_address_program_id: Option<&Pubkey>,
+    compute_unit_price: &Option<u64>,
```

Comment:
I know that we're inconsistent with `&Option<T>` vs `Option<&T>` throughout the cli, but I'd prefer `Option<&u64>` for all these

Reply:
Sounds good to me, make changes one step at a time